index.html

```diff
@@ -32,6 +29,139 @@ <h1 class="company-name">Eleva<span>tion</span></h1>
           <a href="#">Menu <span class="fa fa-bars"></span></a>
         </div>
 
+
+<!--BODY, BUILT BY CMOORE-->
+<body>
+
+<!--SECTION ONE BLOG CONTENT--> 
+<section class="a" id="blog">
```

Comment:
What does the `a` class mean? Can you make this more descriptive so we know what it's for? If it's a placeholder, it's better to remove the `class="a"` code entirely.